### PR TITLE
install: Create OpenSSL CSPRNG seed data for voms-proxy-init.

### PR DESCRIPTION
**Pull request by olifre**
voms-proxy-init tries to open ~/.rnd (via recent openssl libraries)
and if this does not exist, the error:
```
 Cannot open fileFilename=/var/lib/cobald/.rnd
 Function: RAND_load_file
```
is spammed into the cron log output. While it seems the data
is not actually used, best practice to silent the warnings
is to seed it with something useful as described here:
 https://security.stackexchange.com/a/177512

We observe this since the upgrade to CentOS 8.

### CODE REVIEW

**Comment by olifre** [comment]
@wiene , I realized just now that I can't approve my own PR, so I am unsure if the "two approving reviews needed" limitation works out for this PR. If it does not, I would propose we use the `command line instructions` which GitHub links for merging in those cases (better ideas welcome ;-) ).

File: manifests/install.pp
Comment (wiene) [inline manifests/install.pp:323]: The first `of` should be an `if`. Otherwise this PR looks good to me.

**Review by wiene — CHANGES_REQUESTED** [review]

File: manifests/install.pp
Comment (olifre) [reply manifests/install.pp]: Thanks! A second pair of eyes is surely helpful (and of course we should also look at tests later, but only extensive tests actually running the `exec` would have caught this, `puppet-postgresql` is doing such things we might take a look at). 

Should be fixed with the force push from just now :-).

**Review by wiene — APPROVED** [review]

**Comment by olifre** [comment]
Thanks for the quick merging! The change is now already in production :rocket: .

**Pull request merged**
